src/main/resources/hudson/maven/MavenModuleSet/configure-entries.jelly

```diff
@@ -65,7 +65,15 @@ THE SOFTWARE.
       <f:entry title="${%Maven Version}">
         <div class="error">
           ${%Maven Version.error.1}&lt;br>
-          ${%Maven Version.error.2(rootURL)}
+          <!-- In Jenkins 2.x and later, tools configuration is on its own page, not the global configuration. -->
+          <j:choose>
+            <j:when test="${h.version.startsWith('1')">
```

Comment:
:nit: A slightly more robust way would be to check if the configureTools management link exists or not. But then you would either need to write some "backend" code or convert all this jelly to groovy since jelly can't access static fields et. al.